app/views/users/index.html.erb

```diff
@@ -1,31 +1,27 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Users</h1>
+<h1>Grumblrs</h1>
 
 <table>
   <thead>
     <tr>
       <th>Name</th>
       <th>Email</th>
-      <th colspan="3"></th>
+      <th colspan="2"></th>
     </tr>
   </thead>
 
   <tbody>
     <% @users.each do |user| %>
       <tr>
-        <td><%= user.name %></td>
+        <td><%= link_to user.name, user %></td>
         <td><%= user.email %></td>
-        <td><%= link_to 'Show', user %></td>
-        <% if current_user && current_user.id == user.id %>
-          <td><%= link_to 'Edit', edit_user_path(user) %></td>
-          <td><%= link_to 'Destroy', user, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-        <% end %>
```

Comment:
I just figured all this editing and destroying could be done in one place, on the Dashboard